package.json

```diff
@@ -37,6 +37,7 @@
     "grunt-contrib-uglify": "0.8.0",
     "grunt-mocha-test": "0.12.7",
     "grunt-shell-spawn": "0.3.1",
+    "mocha": ">= 1.20.0",
```

Comment:
until now we have done the strict versioning - hence all deps being the actual version but this is a dev-dep and I think we do this (loosey-goosey) in other places. Do you think we should stick with dependencies being the strict version and be loser with the dev-deps? @jebeck thoughts?